app/javascript/react/src/components/Home.jsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+
+const Home = () => {
+  const [data, getData] = React.useState([]);
+  const url = 'http://127.0.0.1:3000/random_greeting';
+  React.useEffect(() => {
+    async function fetchGreeting() {
+      const response = await fetch(url);
+      const result = await response.json();
+      getData(result);
+    }
+    fetchGreeting();
+  }, [url]);
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/connect-frontend-frameworks/rails_react_webpack.md#:~:text=Create%20a%20store%2C%20an%20action%2C%20and%20a%20reducer%20that%20will%20connect%20to%20your%20API%20endpoint%20to%20get%20the%20random%20greeting.), please use redux toolkit and create a store, an action, and a reducer that will connect to your API endpoint to get the random greeting, not directly fetch the data inside `useEffect` as you did.